source/graphics/AnimatedEntity.cpp

```diff
@@ -0,0 +1,69 @@
+#include "../../headers/graphics/AnimatedEntity.hpp"
```

Comment:
Include from root please...
(add `-I./` to GCC command line)

```suggestion
#include "headers/graphics/AnimatedEntity.hpp"
```